CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 -## V 0.2.4
 -*Not yet released*
+- Allow method declaration of `==` and `<` and auto generate `!=`, `>`, `<=` and `>=`.
```

Comment:
It should be in the section below imho, because 0.2.3 has not been released yet.

Reply:
Right